nuxt/components/edit/EditTrackDialog.vue

```diff
@@ -97,14 +97,13 @@
 </template>
 
 <script lang="ts">
-import {
-  Component, Prop, Watch, Vue,
-} from 'nuxt-property-decorator';
+import { Component, Prop, Vue, Watch } from 'nuxt-property-decorator';
 import { clone } from '@/utils/clone';
 import { RequestOptions, TrackIncludes } from '@/api/tracks';
 import TimestampedEditor from '@/components/edit/lyrics/TimestampedEditor.vue';
 import { Documents, Format } from '@/entities/lyrics';
-import JsonV1Document = Documents.JsonV1.Document;
+import { getTrackUri } from '~/entities/track';
+  import JsonV1Document = Documents.JsonV1.Document;
```

Comment:
Not sure why this is indented
```suggestion
import JsonV1Document = Documents.JsonV1.Document;
```